src/main/java/seedu/address/model/person/Memo.java

```diff
@@ -25,7 +25,7 @@
     /**
      * Constructs a {@code Memo}.
      *
-     * @param memo A memo.
+     * @param memo a memo.
```

Comment:
Maybe you should be more descriptive in specifying what is a "memo". Example: "User-supplied argument that is used as a Person's memo" 😄